src/main/java/net/sf/jabref/JabRefPreferences.java

```diff
@@ -455,6 +458,9 @@ private JabRefPreferences() {
         defaults.put(USE_PROXY, Boolean.FALSE);
         defaults.put(PROXY_HOSTNAME, "my proxy host");
         defaults.put(PROXY_PORT, "my proxy port");
+        defaults.put(USE_PROXY_AUTHENTICATION, Boolean.FALSE);
+        //defaults.put(PROXY_USERNAME, "my proxy username");
+        //defaults.put(PROXY_PASSWORD, "my proxy password");
```

Comment:
We don't have to cope with null values because NetworkTab.validateSettings ensures that the properties are neither null nor empty, as long as `useProxyAuthentication` is enabled.